addons/project/views/project_task_type_views.xml

```diff
@@ -36,7 +36,8 @@
                             </group>
                             <group>
                                 <field name="fold"/>
-                                <field name="project_ids" widget="many2many_tags" options="{'color_field': 'color'}"/>
+                                <field name="user_id" invisible="1"/>
+                                <field name="project_ids" widget="many2many_tags" options="{'color_field': 'color'}" attrs="{'required': [('user_id','=', False)]}"/>
```

Comment:
I am not sure I understand why we check the ```user_id``` for the required attribute. Could you please explain why that is ?

Reply:
Cause we don't need to make `project_ids` field required for personal stage and having user set on a stage is a personal stage :slightly_smiling_face:

Reply:
@pkyriakou is right, we cannot really create a personal stage in that form view and so we could directly set `required="1"` instead. 🤔 

```suggestion
                                <field name="project_ids" required="1" widget="many2many_tags" options="{'color_field': 'color'}"/>
```